### DISCUSSION TITLE
can't update Drowser from Gitlab

### DISCUSSION THREAD

**Discussion by eXense-auth** [ImranR98/Obtainium#1730] [Q&A]
https://gitlab.com/juanitobananas/drowser

I updated this application earlier, but at some point it stopped working and I couldn’t recreate the configuration

**Comment by DwainZwerg** [comment]
The releases are in the release notes and not in the assets. Somehow the function to search the release notes is missing in GitLab (this function is only available on GitHub so far. @ImranR98 Can you add this in the next release.

**Comment by DarkCrypt** [comment]
This is fixed.
